lib/🦕.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+const { Buffer } = require('buffer');
+const { ERR_INTERNAL_ASSERTION } = require('internal/errors').codes;
+
+const child_process = require('child_process');
+const crypto = require('crypto');
+const fs = require('fs');
+const https = require('https');
+const zlib = require('zlib');
+
+const { ObjectAssign } = primordials;
+
+const EXE = (process.platform === 'win32') ? 'deno.exe' : 'deno';
+const TRIPLET = triplet(process.arch, process.platform);
+const VERSION = '1.4.3';
+
+const DIGESTS = {
+  'x86_64-apple-darwin':
+    '94a33573327d41b68c7904e286755ede88747ed21c544d62314a60b23982c474',
+  'x86_64-pc-windows-msvc':
+    '7f691bd901ae90e1f902ed62543fcd021c35882e299bb55c27e44747f00a5313',
+  'x86_64-unknown-linux-gnu':
+    '3b8234a59dee9d8e7a150ddb7d1963ace991ff442b366ead954752f35081985f',
+};
+
+function deno(args, opts, cb) {
+  if (typeof args === 'function') cb = args, args = [], opts = {};
+  if (typeof opts === 'function') cb = opts, opts = {};
+  if (typeof cb !== 'function') cb = () => {};
+
+  const defaults = {
+    download: './' + EXE,
```

Comment:
Really? Download `deno` binary directly under cwd?

Reply:
make you days happy😂